src/util/c_types.cpp

```diff
@@ -60,9 +60,9 @@ Function: signed_int_type
 
 \*******************************************************************/
 
-typet signed_int_type()
+signedbv_typet signed_int_type()
 {
-  typet result=signedbv_typet(config.ansi_c.int_width);
+  signedbv_typet result=signedbv_typet(config.ansi_c.int_width);
```

Comment:
`signedbv_typet result(config.ansi_c.int_width);` (and several more instances of the same)